src/index.js

```diff
@@ -23,7 +23,7 @@ import Tab from './components/Tab.react';
 import Store from './components/Store.react';
 import LogoutButton from './components/LogoutButton.react';
 
-import './components/css/react-dates@12.3.0.css';
+import 'react-dates/lib/css/_datepicker.css';
```

Comment:
As before, doing this here because Jest can't process `css` files for some reason that I don't feel like figuring out.